### DISCUSSION TITLE
How to get started?

### DISCUSSION THREAD

**Discussion by nikp86** [eclipse-tractusx/tractusx-edc#1262] [General]
Hi All,

I'm new to the project and want to know the best way/path/steps to get started. So far tried the e2e tutorial and now I'm not sure how to proceed further.

Thank you in advance!
Best Regards,
Nik

**Comment by paullatzelsperger** [comment]
Discussions are intended for specific questions, feature ideas etc. This type of open-ended and unscoped questions have a low chance of getting answered. I will close this discussion, because it's unproductive. Don't expect people to explain to you your own use case.

I recommend you identify your use case, read up on documentation and come back once you have specific questions.
